src/components/pages/user-profile.yml

```diff
@@ -0,0 +1,9 @@
+accountClosed:
+  en: This user account was closed.
+  de: Dieses Benutzerkonto wurde geschlossen.
+documentsHeadline:
+  en: Recent contributions
+  de: Letzte Beiträge
+moreButton:
+  en: Show {count} more
+  de: Zeig {count} weitere an / {count} weitere anzeige[n] ?
```

Comment:
{count} weitere anzeigen

Reply:
But, if `count` is 1 then "{count} weitere**n** anzeigen" (meaning: "1 weiteren Beitrag")

Reply:
Thanks, changed it so.